app/javascript/psulib_blacklight/styles/_external_links.scss

```diff
@@ -8,3 +8,13 @@
   list-style-type: none;
   padding-left: 0;
 }
+
+// scss-lint:disable QualifyingElement
+button.btn.toggle-external-links.collapsed::before {
+  content: 'View More';
+}
+
+button.btn.toggle-external-links::before {
+  content: 'View Less';
+}
+// scss-lint:enable QualifyingElement
```

Comment:
You can actually remove the `button` prefix on these selectors — it's not actually needed, and then you can also remove the `scss-lint` comments. `.btn` is enough to target these elements.

Your selectors can just be `.btn.toggle-external-links.collapsed::before` and `.btn.toggle-external-links::before`

Reply:
ahh thanks! will do this change